test/unit/Git/GetVersionCollectionFromGitRepositoryTest.php

```diff
@@ -76,6 +76,6 @@ public function testFromRepositoryAllowsVersionPrefix() : void
     {
         $this->makeTag('v1.0.0');
 
-        self::assertSame(['1.0.0'], $this->getTags());
+        self::assertSame(['v1.0.0'], $this->getTags());
```

Comment:
This effectively fixes #37, right?

Reply:
I believe so, yes. That should be the scenario you referenced in other comment.